content/advanced-operations/compound-mutiword-expressions.Rmarkdown

```diff
@@ -4,7 +4,7 @@ weight: 20
 draft: false
 ---
 
-We can compound multi-word expressions through collocation analysis. In this example, we identify sequences of capitalized words and compound them as proper names, which are important linguistic features of newspaper articles.
+We can compound multi-word expressions through collocation analysis. In this example, we will identify sequences of capitalized words and compound them as proper names, which are important linguistic features of newspaper articles.
```

Comment:
I agree that it is nice to insert "will" because it is about we are going to do in the example.